content/intro-to-storybook/react/ja/simple-component.md

```diff
@@ -100,17 +96,15 @@ Storybook にコンポーネントを認識させるには、以下の内容を
 コンポーネントにストーリーが複数連なっているので、各ストーリーを単一の `Template` 変数に割り当てるのが便利です。このパターンを導入することで、書くべきコードの量が減り、保守性も上がります。
 
 <div class="aside">
-
-`Template.bind({})` は関数のコピーを作成する [JavaScript の標準的な](https://developer.mozilla.org/ja/docs/Web/JavaScript/Reference/Global_Objects/Function/bind) テクニックで、同じ実装を使いながら、エクスポートされたそれぞれのストーリーに独自のプロパティを設定することができます。
-
+💡 <code>Template.bind({})</code> は関数のコピーを作成する [JavaScript の標準的な](https://developer.mozilla.org/ja/docs/Web/JavaScript/Reference/Global_Objects/Function/bind) テクニックで、同じ実装を使いながら、エクスポートされたそれぞれのストーリーに独自のプロパティを設定することができます。
```

Comment:
@bhbs if you could leave a line between the divs I'd appreciate it. Turning it into:
```md
<div>

💡 <code>Template.bind({})</code> は関数のコピーを作成する [JavaScript の標準的な](https://developer.mozilla.org/ja/docs/Web/JavaScript/Reference/Global_Objects/Function/bind) テクニックで、同じ実装を使いながら、エクスポートされたそれぞれのストーリーに独自のプロパティを設定することができます。

</div>
```

The reasoning behind this is that if you don't include it, it will not render correctly, turning it into.
![render issue](https://user-images.githubusercontent.com/22988955/170719117-b0f15ab1-6bb5-453f-8547-a73b75b1b7ac.png)

Reply:
Oh, that is why there are lines!
Following English ver., I changed markdown to HTML!

commit: a09a468

English ver. :
https://github.com/bhbs/learnstorybook.com/blob/a09a468263b7d62abef34bf9a12bc90f10046c92/content/intro-to-storybook/react/en/simple-component.md?plain=1#L98-L100